python3.html.markdown

```diff
@@ -38,7 +38,7 @@ Note: This article applies to Python 3 specifically. Check out [here](http://lea
 8 - 1   # => 7
 10 * 2  # => 20
 
-# Except division which returns floats, real numbers, by default
+# Except division which returns integers if operating on integers
```

Comment:
since this isn't clear, i'll drop it. but I still think the old line should be deleted because it doesn't make any sense. what about the rest of the changes I suggested?